sys/cloudformation/stack.yaml

```diff
@@ -580,31 +580,6 @@ Resources:
     Properties:
       UserName: !Ref iamusergithubactions
 
-  # REDIS
```

Comment:
One day I'll convert this cloudformation stack into terraform, and then we could import freely any type of resource.